templates/zerver/help/status-and-availability.md

```diff
@@ -98,15 +110,7 @@ unavailable).
 
     You can also use the same menu to mark yourself as available again.
 
-### Disable updating availability
-
-Zulip supports the privacy option of never updating the availability
-information for your account.  The result is that you will always
-appear to other users as **Offline** (or **Unavailable**, if you've
-set an appropriate status), regardless of your activity in Zulip.
-
-With this setting, your "Last active" time displayed to other users in
-the UI will be frozen as the time you enabled this setting.
+#### From personal settings (alternate method on Desktop/Web only)
 
```

Comment:
Instead of this section, let's put these instructions into a tip inside the Desktop/Web tab above. @drrosa may have examples handy of pages where we've done something similar.

Reply:
How about this ...

![Screenshot from 2022-09-23 18-11-40](https://user-images.githubusercontent.com/63245456/192005426-d8b04815-1207-4006-afa6-fe4f3180ab36.png)

Reply:
Yeah that works for me.

Reply:
Works for me!